apps/freeablo/faworld/actor.h

```diff
@@ -62,26 +91,25 @@ namespace FAWorld
                   const std::string& dieAnimPath=""
                   );
 
-            virtual void update(bool noclip, size_t ticksPassed);
+            virtual void update(bool noclip);
```

Comment:
Well, I needed to use world->getCurrentTick elewhere anyway, so I figured it was better to be consistent